e2e-tests/web/test/specs/navigateToSearchResults.e2e.ts

```diff
@@ -24,7 +24,9 @@ describe('navigateToSearchResult', () => {
 
     await expect(results[0]).toExist()
     await expect(results[1]).toExist()
-    expect(await results[1]!.getAttribute('href')).toBe(`/${Routes.dashboard}/language/language-courses`)
+    expect(await results[1]!.getAttribute('href')).toBe(
+      `/${Routes.dashboard}/language/language-courses/online-language-courses`,
+    )
```

Comment:
I think because minisearch has a [different ordering](https://github.com/lucaong/minisearch/blob/72cdfbbe866de860e53de6b37db4534ac8bd1e52/src/MiniSearch.ts#L1270) for search results. It sorts the result by "score", which will change the order of the search result in some cases.

Reply:
Exactly :)